lib/UI/exercise/set_widget.dart

```diff
@@ -3,24 +3,16 @@ import 'package:exerlog/Models/maxes.dart';
 import 'package:exerlog/Models/sets.dart';
 import 'package:exerlog/UI/global.dart';
 import 'package:exerlog/UI/maxes/max_builder.dart';
+import 'package:exerlog/src/utils/logger/logger.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_slidable/flutter_slidable.dart';
 
 class SetWidget extends StatefulWidget {
-  final String name;
-  final Exercise exercise;
-  Function(Exercise, Sets, int) addNewSet;
-  Function(Exercise, Sets, int) removeSet;
-  Function? updateExercise;
-  Function updateTotal;
-  final counter;
-  //Function(Sets, int) createNewSet;
-  final bool isTemplate;
-  int id;
 
   SetWidget(
-      {required this.name,
+      {Key? key,
```

Comment:
same here, maybe we should add this to the linter as well?

Reply:
fixed